src/xunit.netcore.extensions/SkippedTestCase.cs

```diff
@@ -2,16 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace Xunit.NetCore.Extensions
 {
     /// <summary>Wraps another test case that should be skipped.</summary>
-    internal sealed class SkippedTestCase : IXunitTestCase
+    internal sealed class SkippedTestCase : LongLivedMarshalByRefObject, IXunitTestCase
```

Comment:
I missed your comment while I was publishing mine on the main thread. But yes, it exists in both.

https://github.com/dotnet/buildtools/pull/1510#issuecomment-303228025

We just had the same timing haha :)